os-config_arm.yml

```diff
@@ -0,0 +1,328 @@
+hostname: rancher
```

Comment:
I think that's the best way, because think when we'll add support for `arm64`, it gets even worse